db/query_test.go

```diff
@@ -4,9 +4,9 @@ import (
 	"fmt"
 	"testing"
 
+	sgbucket "github.com/couchbase/sg-bucket"
```

Comment:
This change looks inadvertent - could either remove, or at least get rid of the whitespace on line 9.